pwa/src/components/footer/Footer.css

```diff
@@ -0,0 +1,18 @@
+.utrecht-page-footer {
+    display: flex;
+}
+
+.utrecht-page-footer > .utrecht-unordered-list {
+    flex: 1;
+}
+
+.utrecht-page-footer .utrecht-heading-4 {
+    color: white;
+}
+
+.utrecht-page-footer .utrecht-unordered-list__item {
+    --utrecht-icon-size: 20px;
+    display: flex;
+    align-items: center;
+    gap: 5px;
+}
```

Comment:
Git expects an empty line at the end of all files (should automatically be inserted by prettier).